### PR TITLE
Decreases damage of surgical tools

### PR DESCRIPTION
**Pull request by FriendlyContractor**
## About The Pull Request
Decreases damage of surgical tools across the board

## Why It's Good For The Game
Surgical tools are probably the most commonly used implement for validhunting. This PR decreases their damage in attempt to discourage players from validhunting

## Changelog
:cl:
balance: Decreased damage of surgical tools
/:cl:

### CODE REVIEW

**Comment by ToasterBan** [comment]
small error in changelog

"balance: Decreased damage of surgical tools"
would be better

**Comment by St0rmC4st3r** [comment]
8 damage from the circular saw? You do realize the punch damage is 7, right?

**Comment by pro100Player** [comment]
wtf. surg. tools aint that easy to obtain, meanwhile you can craft/print in autolathe more robust items/weapons. Even fire extinguisher deals 10 damage, which is the most common item.

**Comment by Naevii** [comment]
This makes me feel like we should nerf everything that doesdamage to prevent validhunting... 
Its silly thinking.

**Comment by mcmeiler** [comment]
This seems like a salt PR. Also this makes literally every other item better for fighting than a *literal fucking saw*

**Comment by BNGP0** [comment]
If a surgical saw is as strong as your punch, then you should be able to saw bones with bare hands without any slowdown or risk during surgery

File: code/modules/surgery/tools.dm [hunk 1]
```diff
@@ -105,7 +105,6 @@
 	hitsound = 'sound/weapons/circsawhit.ogg'
 	materials = list(/datum/material/iron=10000, /datum/material/glass=6000)
 	flags_1 = CONDUCT_1
-	force = 15
```
Comment (PowerfulBacon) [inline code/modules/surgery/tools.dm:108]: For reference 15 damage is the damage of a welder when lit. Would consider lowing this one to toolbox damage at worst (12)

Comment (PowerfulBacon) [reply code/modules/surgery/tools.dm:108]: Also isn't this going to be reducing it to 0 damage?

File: code/modules/surgery/tools.dm [hunk 2]
```diff
@@ -141,7 +139,7 @@
 	righthand_file = 'icons/mob/inhands/equipment/medical_righthand.dmi'
 	item_state = "scalpel"
 	flags_1 = CONDUCT_1
-	force = 10
+	force = 7
```
Comment (PowerfulBacon) [inline code/modules/surgery/tools.dm:142]: For reference 7 damage is the amount of damage caused by a punch. Toolboxes have 12 damage and welders have 15.

File: code/modules/surgery/tools.dm [hunk 3]
```diff
@@ -163,7 +161,7 @@
 	icon = 'icons/obj/surgery.dmi'
 	icon_state = "scalpel"
 	flags_1 = CONDUCT_1
-	force = 10
+	force = 7
```
Comment (PowerfulBacon) [inline code/modules/surgery/tools.dm:164]: Again, note that 7 = punching damage

File: code/modules/surgery/tools.dm [hunk 4]
```diff
@@ -189,7 +187,7 @@
 	hitsound = 'sound/weapons/circsawhit.ogg'
 	throwhitsound =  'sound/weapons/pierce.ogg'
 	flags_1 = CONDUCT_1
-	force = 15
+	force = 8
```
Comment (PowerfulBacon) [inline code/modules/surgery/tools.dm:190]: 15 is not a bad weapon, same damage as a welding tool and 3 more than a toolbox. 8 is way too low imo

File: code/modules/surgery/tools.dm [hunk 5]
```diff
@@ -315,7 +313,7 @@
 	icon = 'icons/obj/surgery.dmi'
 	icon_state = "scalpel_a"
 	hitsound = 'sound/weapons/blade1.ogg'
-	force = 16
+	force = 9
```
Comment (PowerfulBacon) [inline code/modules/surgery/tools.dm:316]: First one that is actually better than easilly obtainable station items

File: code/modules/surgery/tools.dm (hunk 1 above)
Comment (PowerfulBacon) [reply code/modules/surgery/tools.dm:108]: (which will also mean no damage taken if you fail surgery)

**Review by PowerfulBacon — COMMENTED** [review]
If the problem is validhunting, all the others do less damage than a toolbox. The problem with validhunting can't be fixed by changing the items, you will only get the validhunters using a different item of equal value (Only the laser scalpel is better than a welder, surgical drill and the circular saw are the only things better than a toolbox with equal damage to a welding tool).

Note that this also affects damage taken when you fail surgery.

While I am partly against changing these values for validhunting reasons, since they aren't particularly *that* good, some items I agree need to be made more sense when it comes to forces (some items do more damage than things that don't make sense for them to in the game as a whole)

**Comment by FriendlyContractor** [comment]
Overwhelming negative feedback. I wonder what you use those tools for if decrasing their damage is such a problem

**Pull request closed without merging**